src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl

```diff
@@ -2144,7 +2144,8 @@
           },
           "bundleName": {
             "description": "Bundle name",
-            "type": "string"
+            "type": "string",
+            "deprecated": true
```

Comment:
Update the description with that field to use, e.g. "DEPRECATED: Use `pspBusinessName` instead."